pkg/sdn/api/validation/validation.go

```diff
@@ -87,7 +87,10 @@ func ValidateHostSubnet(hs *sdnapi.HostSubnet) field.ErrorList {
 
 	_, _, err := net.ParseCIDR(hs.Subnet)
 	if err != nil {
-		allErrs = append(allErrs, field.Invalid(field.NewPath("subnet"), hs.Subnet, err.Error()))
+		// check if annotation exists, then let the Subnet field be empty
+		if _, ok := hs.Annotations["hostsubnet.sdn.openshift.io/autocreate"]; !ok {
```

Comment:
This annotation is mainly to address F5 use case. We use actual fields for generic use cases?